### PR TITLE
UICIRC-647: Add Jest/RTL tests for `RequestPolicyForm` component

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add RTL/Jest testing for `RequestPolicyForm` component in `src/settings/RequestPolicy`.

## Refs
https://issues.folio.org/browse/UICIRC-647

## Screenshots
image

### CODE REVIEW

**Review by pavelspichonak — APPROVED** [review]

File: src/settings/RequestPolicy/RequestPolicyForm.test.js [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+      }), {});
+    });
+
+    it('"CancelButton" should be executed with correct props', () => {
+      expect(CancelButton).toHaveBeenCalledWith({
+        onCancel,
+      }, {});
+    });
+
+    it('"CancelButton" should be executed with correct props', () => {
```
Comment (artem-blazhko) [inline src/settings/RequestPolicy/RequestPolicyForm.test.js:135]:
```suggestion
    it('"FooterPane" should be executed with correct props', () => {
```

File: src/settings/RequestPolicy/RequestPolicyForm.test.js [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  children,
+  firstMenu,
+  footer,
+}) => (
+  <div>
+    {firstMenu}
+    {children}
+    {footer}
+  </div>
+));
```
Comment (artem-blazhko) [inline src/settings/RequestPolicy/RequestPolicyForm.test.js:43]: It is redundant. We already have the same mock here - https://github.com/folio-org/ui-circulation/blob/e6f072c3e32ac577021692c3a4d6cefc51356127/test/jest/__mock__/stripesComponents.mock.js#L88

File: src/settings/RequestPolicy/RequestPolicyForm.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from 'react-intl';
+import { injectIntl } from 'react-intl';
```
[the remark this answers is not in the document]
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/RequestPolicyForm.js:3]: As <FormattedMessage> renders a component, we can't check it on props passing, and must render it directly to check `id`.

File: src/settings/RequestPolicy/RequestPolicyForm.test.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/RequestPolicyForm.test.js]: Nice point, done.

File: src/settings/RequestPolicy/RequestPolicyForm.test.js (hunk 2 above)
Comment (Stanislau-Snapok) [reply src/settings/RequestPolicy/RequestPolicyForm.test.js]: You are right! Thanks!

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**